Sources/Jobs/JobsConfiguration.swift

```diff
@@ -25,6 +29,21 @@ public struct JobsConfig {
         storage[key] = job
     }
     
+    /// Schedules a new job for execution at a later date.
+    /// Example: `config.schedule(job).daily().at(.startOfDay)`
```

Comment:
Examples should be on a new line and indented to create a code block in the doc blocks

Reply:
Will do!